.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,9 +19,10 @@ How to reproduce the issue. Please include a minimum working code sample, the
 traceback (if any) and the anonymized DICOM dataset (if relevant).
 
 **Your environment**
-Please run the following and paste the output.
-```bash
-$ python -c "import platform; print(platform.platform())"
-$ python -c "import sys; print('Python ', sys.version)"
-$ python -c "import pydicom; print('pydicom ', pydicom.__version__)"
-```
+Please run the following code snippet and paste the output.
+```python
+import platform, sys, pydicom
+print(platform.platform(),
+      "\nPython", sys.version,
+      "\npydicom", pydicom.__version__)
```

Comment:
I switched to 1 print statement from 3 to avoid having the `>>>` s in the output. That's more for the maintainers than the submitter, but I thought it made the results easier to read.

I'd originally put the print on 1 line, but it got up over 80 chars, so I broke it for better rendering.

Reply:
I toyed a little bit with something like 
```python
import platform
import sys
import importlib

output = [
    platform.platform(),
    "Python " + sys.version,
]

for module in ("pydicom", "numpy", "PIL", "imagecodecs", "gdcm"):
    try:
        m = importlib.import_module(module)
        output.append(module + " " + m.__version__)
    except:
        pass

print("\n".join(output))
```

(in https://gist.github.com/blairconrad/37888634031016865dec93c33ff7b48f), but it's a little heavy, unless you run it from the gist (or other file in the pydicom repo), but that's got its own problems…)

Reply:
> it's a little heavy, unless you run it from the gist (or other file in the pydicom repo), but that's got its own problems…)

Why not make it a file in the pydicom distribution?  I think that could be really useful, might even prevent an issue from being created if people can see that a library is not the version they expected.  

We would still need something in the issue template for a while for those who hadn't updated to pydicom >= 2, but eventually that could become *the* way to gather the info.  "Please run `python -m pydicom.env_info`" (we could work on the name). 

I think every library should be listed though, not just the found ones.  So absence should be indicated:  `gdcm: not installed`, etc.

> If I've crossed the line from "helpful contributor" to "annoying pest", just say. I'm a package maintainer too. I'll back off and not be offended.

Don't worry, you are not there yet.  :wink:

Reply:
Thanks, @darcymason.

> I think every library should be listed though, not just the found ones. So absence should be indicated: gdcm: not installed, etc

Haha. For fun, I'd actually continued working on the script, and had this sitting around:

```python
import platform
import sys
import importlib

tidbits = [("platform", platform.platform()), ("Python", sys.version)]

for module in ("pydicom", "numpy", "PIL", "jpeg_ls", "gdcm"):
    try:
        m = importlib.import_module(module)
        tidbits.append((module, m.__version__))
    except:
        tidbits.append((module, "not found"))

print(
    "| module | version |\n| --- | --- |\n| "
    + (" |\n| ".join((module + " | " + version for (module, version) in tidbits)) + " | ")
)
```

Output is 
```markdown
| module | version |
| --- | --- |
| platform | Windows-10-10.0.18362-SP0 |
| Python | 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 23:09:28) [MSC v.1916 64 bit (AMD64)] |
| pydicom | 2.0.0.dev0 |
| numpy | 1.18.3 |
| PIL | 7.1.2 |
| jpeg_ls | not found |
| gdcm | not found |
```

which renders as
| module | version |
| --- | --- |
| platform | Windows-10-10.0.18362-SP0 |
| Python | 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 23:09:28) [MSC v.1916 64 bit (AMD64)] |
| pydicom | 2.0.0.dev0 |
| numpy | 1.18.3 |
| PIL | 7.1.2 |
| jpeg_ls | not found |
| gdcm | not found |

I can look into adding it as part of the distro, if you're so inclined. Of course, that means there's a lag while people upgrade to the distribution that has the tool (including those Python 2.xers, like you said.)

Reply:
> I can look into adding it as part of the distro, if you're so inclined.

Yes, please.  I really like this - a nice summary table like that is a good debug tool for users, and easy for us to absorb the critical info in an issue.

One coding suggestion, though, to future-proof in case an import module does not have `__version__`, maybe separate the try: except: to just work on the import itself, use an `else:` clause in the try block for the `__version__`.